e2e/snapshot_ops_test.go

```diff
@@ -0,0 +1,234 @@
+package e2e
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/gluster/glusterd2/e2e/lvmtest"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/restclient"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	snapname = "snaptest"
+)
+
+// TestSnapshot creates a volume and snapshots, runs further tests on it and
+// finally deletes the volume
+func TestSnapshot(t *testing.T) {
+	var err error
+	var brickPaths []string
+	r := require.New(t)
+
+	gds, err = setupCluster("./config/1.toml", "./config/2.toml")
+	if err != nil {
+		r.Nil(err)
+		teardownCluster(gds)
+		return
+	}
+	defer teardownCluster(gds)
+
+	client = initRestclient(gds[0].ClientAddress)
+	brickPaths, err = lvmtest.CreateLvmBricks(4)
+	if err != nil {
+		r.Nil(err)
+		lvmtest.CleanupLvmBricks(4)
+		return
+	}
+	defer func() {
+		err := lvmtest.CleanupLvmBricks(4)
+		r.Nil(err)
+	}()
+
+	// Create the volume
+	if err := volumeCreateOnLvm(brickPaths, client); err != nil {
+		t.Logf("Failed to create the volume")
```

Comment:
remove logging and return. if you want to log you can use  
```
r.Nil(err,"message to print if it fails")
```
take care  of this in rest of the code